Classes/InCallViewControllerNew.m

```diff
@@ -0,0 +1,525 @@
+//
+//  InCallViewControllerNew.m
+//  linphone
+//
+//  Created by Hrachya Stepanyan on 3/1/16.
```

Comment:
This class does not seem to support rotation landscape in the UI on iPhone 5. Please allow rotation to the video and image assets.